src/main/java/seedu/address/model/person/Name.java

```diff
@@ -10,7 +10,8 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank.";
+            "Names should only contain alphanumeric characters and spaces, and it should not be blank."
+                    + "\nSpecial characters like -, \', and / are supported.";
```

Comment:
Rephrasing to match the user guide
```suggestion
            "Name must not be blank, and it must contain only alphanumeric characters, spaces,"
                    + " slashes, and dashes.";
```